app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,62 @@
+class AlbumsController < ApplicationController
+  before_action :get_album, only: [:edit, :update, :destroy]
+  def get_album
+    @album = Album.find(params[:id])
+  end
+
+  def index
+    @albums = Album.all
+  end
+
+  def new
+    @url = "create"
```

Comment:
You have this included in the books and movies controllers as well but it doesn't appear that it is being used anywhere.